client/jetpack-cloud/sections/partner-portal/primary/licenses/index.tsx

```diff
@@ -34,12 +46,37 @@ export default function Licenses( {
 	sortField,
 }: Props ): ReactElement {
 	const translate = useTranslate();
+	const dispatch = useDispatch();
+	const partner = useSelector( getCurrentPartner );
+	const activeKeyId = useSelector( getActivePartnerKeyId );
+	const onKeySelect = useCallback( ( option ) => {
+		dispatch( setActivePartnerKey( parseInt( option.value ) ) );
+	}, [] );
+
+	const options =
+		partner &&
+		partner.keys.map( ( key ) => ( {
+			value: key.id.toString(),
+			label: key.name,
+		} ) );
+
+	options?.unshift( { label: translate( 'Partner Key' ) as string, value: '', isLabel: true } );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **7** times:
`translate( 'Partner Key not found' )` *ES Score: 8*